src/Resources/doc/configuration-reference.md

```diff
@@ -0,0 +1,116 @@
+# Configuration Reference
+
+``` yaml
+eight_points_guzzle:
+    # (de)activate logging/profiler; default: %kernel.debug%
+    logging: true
+
+    clients:
+        api_payment:
```

Comment:
I've also seen some packages that have a `minimum workable config", maybe that would be nice to add here as well. For example, in a project where I use this I just have this:

```
eight_points_guzzle:
  clients:
    api_payment: ~
```

and that works just fine. Just to show you don't _have_ pass any parameters if you don't want to.

Reply:
Good idea :+1: did it.